src/resources/platforms/index.tsx

```diff
@@ -40,8 +40,15 @@ const PlatformForm = (): React.ReactElement => {
 
 const PlatformCreate = ({ name }: PropType): React.ReactElement => {
   const cName: string = name
+  const redirect = useRedirect()
   return (
-    <Create redirect={`/${cName}`} resource={constants.R_PLATFORMS}>
+    <Create
+      mutationOptions={{
+        onSuccess: (data: { platformNumber: string; id: number }): void => {
```

Comment:
Hello @TahaKhanAbdalli - on line 48 we direct to `cName`.  In the other changes we direct to `constants.R_xxxx`.

Is there a reason why this module is different?

Reply:
It is in reference items; that's why we use `cName`. But now I have changed the `cName` with `constants.R_PLATFROM`. It is working fine with both approaches; there is no issue.